src/components/AppLayout.jsx

```diff
@@ -57,48 +59,61 @@ const AppLayout = () => {
   }
 
   return (
-    <Layout>
-      <Sidebar>
-        <Nav>
-          <NavItem>
-            <NavLink {...makeProps(['timetable'])}>
-              <NavIcon icon={CalendarIcon} />
-              <NavText>{t('Sidebar.timetable')}</NavText>
-            </NavLink>
-          </NavItem>
-          <NavItem>
-            <NavLink {...makeProps(['homeworks'])}>
-              <NavIcon icon={CheckboxIcon} />
-              <NavText>{t('Sidebar.homeworks')}</NavText>
-            </NavLink>
-          </NavItem>
-          <NavItem>
-            <NavLink {...makeProps(['grades'])}>
-              <NavIcon icon={PieChartIcon} />
-              <NavText>{t('Sidebar.grades')}</NavText>
-            </NavLink>
-          </NavItem>
-          <NavItem>
-            <NavLink {...makeProps(['presence'])}>
-              <NavIcon icon={WalkIcon} />
-              <NavText>{t('Sidebar.presence')}</NavText>
-            </NavLink>
-          </NavItem>
-        </Nav>
-      </Sidebar>
-      <BarComponent />
-      <Main>
-        <Content>
-          {isMobile && (
-            <BarCenter>
-              <Typography variant="h5">{client.appMetadata.slug}</Typography>
-            </BarCenter>
+    <AccountProvider>
```

Comment:
nit: Extract this call inside `AppProviders` so its collocated with all the providers ([code](https://github.com/cozy/cozy-papillon-app/blob/e752499508f3c764a61e3235186c79c3ba283698/src/components/AppProviders.jsx))

Reply:
Extracted here -> https://github.com/cozy/cozy-papillon-app/pull/29/commits/64ea6cfcf1f3c714f1c1fea7ff7a4d88f781a26a